docs/release/trg-7/trg-7-09.md

```diff
@@ -0,0 +1,79 @@
+---
+title: TRG 7.09 - Deprecation of Unmaintained Product Repositories
+---
+
+| Status | Created     | Post-History                          |
+|--------|-------------|---------------------------------------|
+| Draft  | 02-Oct-2024 | Initial draft                         |
+
+## Why
+
+Eclipse Tractus-X is an open source project governed by the Eclipse Foundation and its Committers. Beside the role participating in maintaining the code base, within Eclipse Tractus-X the group of committers also acts as the project's security team. They have access to the tab "Security" and may see the vulnerabilities detected by the tooling set up in [TRGs 8](../trg-8/trg-8-01.md). As this group of committers is responsible for the whole project and has limited resources, projects that don't receive any updates for a given time period will be deprecated. This is a "value-for-money" decision.
+
+## Description
+
+Deprecation applies to repositories of type `product`. This information is maintained in the repository metafile (see [TRG 2.05](../trg-2/trg-2-5.md)).Deprecation of a repository means that **the repository is beeing archived**. It's still accessible from within the project, so that it could be revived. Please refer to [this list for examples of archived repositories within Eclipse Tractus-X](https://github.com/eclipse-tractusx/?q=&type=archived&language=&sort=).
+
+Additionally the helm charts and container images are still available and no information about the deprecation is passed to the user.
+
+### Criteria for Deprecation
+
+Committers consider the following criteria to propose deprecation of a repository:
+
+1. The product of the repository has not been released within the *past year's major release cycle* **\***.
+2. The standards used by the product have been changed in a major version within the *past year's major release cycle* **\***.
```

Comment:
```suggestion
1. The repository does not comply with the Tractus-X Releases Guidelines (TRG's) and/or the guidelines set by the Eclipse Foundation (i.e. Security, Intellectual Property, etc.)

In case of repositories belonging to a [Product](https://eclipse-tractusx.github.io/community/products):
2. The product of the repository has not been released within the *past year's major release cycle* **\***.
3. The standards used by the product have been changed in a major version within the *past year's major release cycle* **\***.
```